deployer-image/Makefile

```
@@ -4,18 +4,16 @@ MPDEV := $(BIN_DIR)/mpdev
-RELEASE ?= ${TRACK}.0
+TRACK ?= 4.1
+RELEASE ?= ${TRACK}.2
 
 # Docker registry and image names
 REGISTRY = gcr.io/stackgen-gcp-marketplace
 APP_NAME = stackgen-deployer
 DEPLOYER_IMAGE_RELEASE := $(REGISTRY)/$(APP_NAME)/deployer:$(RELEASE)
 DEPLOYER_IMAGE_TRACK := $(REGISTRY)/$(APP_NAME)/deployer:$(TRACK)
 
-.PHONY: all push setup doctor install verify validate clean help
+.PHONY: all push setup doctor install install-crd verify validate clean help
```

checkmake (0.2.2)

[warning] Missing required phony target "test"

(minphony)